docs/advanced/quarantine-workflow.md

```
@@ -0,0 +1,102 @@
+| Your worker fleet can run the scan cron | You don't have a queue / scheduler set up |
+| Scanner quotas / costs make per-request scanning impractical | Scanner cost is irrelevant |
+
+You can switch per-call too:
+
+```php
+$post->attachSecureFile($file, ['async' => true]);
+```
+
+## The flow
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)